core/src/main/kotlin/org/evomaster/core/problem/security/service/SSRFAnalyser.kt

```diff
@@ -101,25 +110,23 @@ class SSRFAnalyser {
         // The below steps are generic, for future extensions can be
         // accommodated easily under these common steps.
 
+        if (httpCallbackVerifier.isActive) {
+            // Reset before execution
+            httpCallbackVerifier.reset()
+        } else {
+            httpCallbackVerifier.prepare()
+        }
+
         // Classify endpoints with potential vulnerability classes
         classify()
 
-        if (actionVulnerabilityMapping.isNotEmpty()) {
-            if (httpCallbackVerifier.isActive) {
-                // Reset before execution
-                httpCallbackVerifier.resetHTTPVerifier()
-            } else {
-                httpCallbackVerifier.initWireMockServer()
-            }
-        }
-
         // execute
         analyse()
 
         // TODO: This is for development, remove it later
         val individualsAfterExecution = RestIndividualSelectorUtils.findIndividuals(
             this.archive.extractSolution().individuals,
-            statusCodes = listOf(200, 201)
+            statusCodes = listOf(200, 201, 204)
```

Comment:
isn't this duplicating what at line 102? if really need, should refactor out in a common constant list to reuse